app/frontend/users/UserEdit.res

```diff
@@ -7,6 +7,8 @@ type state = {
   name: string,
   about: string,
   locale: string,
+  email: string,
+  disable_update_email: bool,
```

Comment:
The variable name is confusing, a suggestion would be `render_email_editor`? (not sure)